scripts/myfans_dl.py

```
@@ -31,15 +31,34 @@ def DL_File(m3u8_url_download, output_file, input_post_id):
+            ["ffmpeg", "-n", "-i", m3u8_url_download, "-c", "copy", "-loglevel", "quiet", ts_file],
             check=True,
             stdout=subprocess.PIPE,
             stderr=subprocess.PIPE,
-            timeout=30  
+            timeout=600  # 10 minutes for download
         )
 
-        return True
+        # Step 2: Convert to MP4 if download successful
+        if os.path.exists(ts_file):
+            convert_result = subprocess.run(
```

Ruff (0.8.2)

Local variable `convert_result` is assigned to but never used

Remove assignment to unused variable `convert_result`

(F841)